sklearn/cluster/tests/test_k_means.py

```diff
@@ -759,31 +802,15 @@ def test_k_means_init_centers():
                                                 init_centers))
 
 
-def test_sparse_k_means_init_centers():
-    from sklearn.datasets import load_iris
-
-    iris = load_iris()
-    X = iris.data
-
+@pytest.mark.parametrize("data", [X, X_csr], ids=["dense", "sparse"])
+def test_k_means_init_fitted_centers(data):
```

Comment:
Here we are changing the test dataset from iris to `make_blobs`. I suppose it's shouldn't matter too much..